installer.py

```diff
@@ -64,7 +64,7 @@ def run(region, domain, account_id, install_server):
       'prompt':"Installing Plugin...",
       'commands':[
         'pip3 install --user .',
-        'aws configure set plugins.bolt awscli-plugin-bolt'
+        'aws configure set plugins.granica awscli-plugin-granica'
       ]
     },
 
```

Comment:
I see some command below:

aws --profile={} configure set bolt.url https://bolt.{}'.format(region,domain

Reply:
Does it become quicksilver.

Reply:
Does it become quicksilver.

Reply:
This variable is not needed.